contract_api/lambda_handler.py

```diff
@@ -1,136 +1,91 @@
-        elif path[0:14] in ["/organizations"] or path[0:5] in ["/tags"] or path == "/search":
-            obj_srch = Search(obj_repo=db[net_id])
-
-        if "/service" == path:
-            data = obj_srvc.get_curated_services()
-        elif "/channels" == path:
-            data = get_profile_details(user_address=payload_dict['user_address'], obj_srvc=obj_srvc)
-        elif "/feedback" == path and event['httpMethod'] == 'GET':
-            data = get_user_feedback(payload_dict['user_address'], obj_srvc=obj_srvc)
-        elif "/feedback" == path and event['httpMethod'] == 'POST':
-            data = set_user_feedback(payload_dict['feedback'], obj_srvc=obj_srvc, net_id=net_id)
-        elif "/group-info" == path:
-            data = obj_srvc.get_group_info()
-        elif "/available-channels" == path:
-            data = obj_chnnl.get_channel_info(payload_dict['user_address'], payload_dict['service_id'],
-                                              payload_dict['org_id'])
-        elif "/expired-channels" == path:
-            data = obj_chnnl.get_expired_channel_info(payload_dict['user_address'])
-        elif "/search" == path:
-            data = obj_srch.get_srch_rslt(payload_dict)
-        elif "/organizations" == path:
-            data = {"organizations": obj_srch.get_all_org()}
-        elif re.match("^(/organizations)[/][[a-z0-9]+$", path):
-            data = obj_srch.get_org(org_id=event['path'].split("/")[2])
-        elif re.match("^(/organizations)[/][a-zA-Z0-9]{1,}[/](services)$", path):
-            data = {"services": obj_srch.get_all_srvc(org_id=event['path'].split("/")[2])}
-        elif re.match("^(/organizations)[/][a-zA-Z0-9]{1,}[/](services)[/][a-z0-9]+$", path):
-            data = []
-        elif re.match("^(/tags)[/][[a-z0-9]+$", path):
-            data = {"services": obj_srch.get_all_srvc_by_tag(tag_name=event['path'].split("/")[2])}
-        else:
-            return get_response(500, "Invalid URL path.")
-        if data is None:
-            err_msg = {'status': 'failed', 'error': 'Bad Request', 'api': event['path'], 'payload' : payload_dict}
-            obj_util.report_slack(1, str(err_msg))
-            response = get_response("400", err_msg)
+        method = event['httpMethod']
+        response_data = None
+
+        if method == 'POST':
+            payload_dict = json.loads(event['body'])
+        elif method == 'GET':
+            payload_dict = event.get('queryStringParameters')
         else:
-            response = get_response("200", {"status": "success", "data": data})
-    except Exception as e:
-        err_msg = {"status": "failed", "error": repr(e), 'api': event['path'], 'payload': payload_dict}
-        obj_util.report_slack(1, str(err_msg))
-        response = get_response(500, err_msg)
-        traceback.print_exc()
+            return get_response(405, "Method Not Allowed")
 
-    return response
+        if path.split("/")[1] in ["/org", "/service"]:
```

Comment:
Please see my comments on the original PR regarding splitting path each time in the if clause